Liubou.Zviaruha/L2/index.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<html>
+
+<head>
+  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <link href="index.css" rel="stylesheet" type="text/css">
+  <link href="normalize.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Mono" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Crete+Round" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Acme" rel="stylesheet">
+  <title>Playground</title>
+</head>
+
+<body class="body">
```

Comment:
body is a single thing, so better just use `body {}` selector and not a class :)